cwl/src/main/scala/wdl4s/cwl/ArgumentToCommandPart.scala

```diff
@@ -2,18 +2,19 @@ package wdl4s.cwl
 
```

Comment:
before we get started... are we sure this should target develop?

Reply:
I think so yes, at least we want artifacts so we can easily reference them in Cromwell for the womification.
For this particular PR things will change but it doesn't break anything and adds stuff we need for WOM in Cromwell so I think we can merge to develop.